mkosi/__init__.py

```diff
@@ -4277,6 +4305,24 @@ def parse_args_file_group(argv_post_parsed: List[str], default_path: str) -> Com
     return parser.parse_args(argv_post_parsed, CommandLineArguments())
 
 
+def parse_args_distribution_group(argv_post_parsed: List[str], default_path: str, distribution: str) -> CommandLineArguments:
+    all_defaults_files = []
+    defaults_dir = default_path + ".d"
+
+    distribution_dir = os.path.join(defaults_dir, distribution)
+    if os.path.isdir(distribution_dir):
+        for distribution_file in sorted(os.listdir(distribution_dir)):
+            distribution_path = os.path.join(distribution_dir, distribution_file)
+            if os.path.isfile(distribution_path):
+                all_defaults_files.append(ArgumentParserMkosi.fromfile_prefix_chars + distribution_path)
+
+    # Insert the distro specific config files after the rest of the config files so they override these.
+    i = next(i for i, v in enumerate(argv_post_parsed) if not v.startswith('@'))
```

Comment:
Should the `@` here be a literal @ or should this be
```suggestion
    i = next(i for i, v in enumerate(argv_post_parsed) if not v.startswith(ArgumentParserMkosi.fromfile_prefix_chars))
```

Reply:
But just in general, I find this line a bit hard to read. It makes perfect sense and I think it's the best performing way to write this, but whenever I see `next` I need to think about the intent for a second.

This is equivalent to (combined with the line below)
```python
for i, v in enumerate(argv_post_parsed):
    if not v.startswith('@'):
        argv_post_parsed[i:i] = all_defaults_files
        break
```
is it?